src/patternfly/components/Nav/nav.scss

```diff
@@ -707,6 +797,19 @@
   & + & {
     --pf-c-nav__section--MarginTop: var(--pf-c-nav__section--section--MarginTop);
   }
+
+  &.pf-m-flyout {
+    --pf-c-nav__section-title--Color: var(--pf-c-nav__section--m-flyout__section-title--Color);
```

Comment:
Currently these styles that change the title color, border, and space this section properly relative to nav items above/below it are tied to a `pf-m-flyout` variation. Is there a use case for this style of creating a group of nav links adjacent to other non-grouped nav links that isn't a flyout menu? Meaning they're just regular links.

How it is currently:

<img width="589" alt="Screen Shot 2021-08-18 at 10 45 20 AM" src="https://user-images.githubusercontent.com/35148959/129929415-6eec32a1-9977-4c9b-a4d5-c2fb677de414.png">

What I'm referring to:

<img width="452" alt="Screen Shot 2021-08-18 at 10 46 19 AM" src="https://user-images.githubusercontent.com/35148959/129929578-ceea6bb5-a34a-4e6b-b15f-706415a151e5.png">

I'm just wondering if we should make this style of group a more generic class instead of tying it to `pf-m-flyout`, assuming someone wants to do what's in the second screenshot. wdyt @mcarrano